src/containers/Main/VrtConversion.tsx

```diff
@@ -0,0 +1,208 @@
+
+  // account
+  const { account } = useWeb3React();
+  const { fastRefresh } = useRefresh();
+
+  // contracts
+  const vrtConverterContract = useVrtConverter();
+  const xvsVestingContract = useXvsVesting();
+  const vrtTokenContract = useVrtToken();
+  const xvsTokenContract = useToken('xvs');
+
+  useEffect(() => {
+    let mounted = true;
+    const update = async () => {
+      const [
+        {
+          redeemableAmount: redeemableAmountTemp,
+          dailyUtilisation: dailyUtilisationTemp,
+          vrtDailyLimit: vrtDailyLimitTemp,
+          numberOfDaysSinceStart: numberOfDaysSinceStartTemp,
+        },
+        conversionRatioTemp,
+        conversionEndTimeTemp,
+        userVrtBalanceTemp,
+        userVrtAllowanceTemp,
+        vrtConverterXvsBalanceTemp,
+      ] = await Promise.all([
+        vrtConverterContract.methods
+          .computeRedeemableAmountAndDailyUtilisation()
+          .call(),
+        // fetch infos
+        vrtConverterContract.methods.conversionRatio().call(),
+        vrtConverterContract.methods.conversionEndTime().call(),
+        account
+          ? vrtTokenContract.methods.balanceOf(account).call()
+          : Promise.resolve(0),
+        account
+          ? vrtTokenContract.methods
+            .allowance(account, getVrtConverterAddress())
+            .call()
+          : Promise.resolve(0),
+        xvsTokenContract.methods
+          .balanceOf(vrtConverterContract.options.address)
+          .call(),
+      ]);
+      if (mounted) {
+        setLoading(false);
+        setRedeemableAmount(new BigNumber(redeemableAmountTemp).div(VRT_DECIMAL));
+        setDailyUtilisation(new BigNumber(dailyUtilisationTemp).div(VRT_DECIMAL));
+        setVrtDailyLimit(new BigNumber(vrtDailyLimitTemp).div(VRT_DECIMAL));
+        setMumberOfDaysSinceStart(new BigNumber(numberOfDaysSinceStartTemp));
+        setConversionRatio(
+          new BigNumber(conversionRatioTemp).div(CONVERSION_RATIO_DECIMAL),
+        );
+        setConversionEndTime(new BigNumber(conversionEndTimeTemp)); // in seconds
+        setUserVrtBalance(new BigNumber(userVrtBalanceTemp).div(VRT_DECIMAL));
+        setUserEnabled(new BigNumber(userVrtAllowanceTemp).gt(0));
+        setVrtConverterXvsBalance(
+          new BigNumber(vrtConverterXvsBalanceTemp).div(XVS_DECIMAL),
+        );
+      }
+    };
+
+    update();
+
+    return () => {
+      mounted = false;
+    };
+  }, [fastRefresh, account]);
+
+  return (
+    <MainLayout title="Redeem VRT">
+      {loading ? <LoadingSpinner /> : (
+        <VrtConversionWrapper>
+          <Row className="vrt-conversion-container">
+            <Col
+              xl={{ span: 8, offset: 8 }}
+              lg={{ span: 12, offset: 6 }}
+              md={{ span: 12, offset: 6 }}
+              sm={{ span: 24 }}
+              xs={{ span: 24 }}
+            >
+              <div className="container">
+                <TabContainer
+                  styles={{
```

Comment:
I think as a policy we should avoid inline styles because they can't be overridden

Reply:
what's the proposed way for users to add customized styles though? I think users already explicitly set the styles so they should have the highest priority

Reply:
The preferred way would be to use a class name. This provides a couple advantages it keeps all styles. It keeps all styles organized together and it allows for the styles to be overridden easier.

For example, we might find ourselves in a situation where the parent wants to customize the styles of a child dynamically but if that style is passed in with the style attribute we won't be able to do that.

Reply:
do you mean something like `<TabContainer className={props.className} />` ?

Reply:
or we can just get rid of style customization here, because if we want to design more universal components, this should be more well designed

Reply:
event something like this: `<TabContainer className={classNames(styles.tabContainer, props.className)} />`